cluster/ubuntu/build.sh

```diff
@@ -40,7 +40,7 @@ cp flannel-${FLANNEL_VERSION}/flanneld binaries/minion
 
 # ectd
 echo "Download etcd release ..."
-ETCD_VERSION="v2.0.9"
+ETCD_VERSION="v2.0.12"
```

Comment:
Note that we have not qualified 2.0.12 ourselves.  I don't know if we want to STOP you from doing this, but you have to know that any problems you hit, we can't help yet.

@lavalamp @dchen1107 how aggressively do we want to discourage this?